samples/js/js-eventsourced-shopping-cart/proto/shoppingcart_api.proto

```diff
@@ -53,6 +53,18 @@ message GetShoppingCart {
 // end::messages2[]
 // tag::service[]
 service ShoppingCartService {
+// end::service[]
+  // tag::proto_state[]
+  // This entity is related to the domain by `state` and `events`
+option (kalix.codegen).event_sourced_entity = {
+    name: "ShoppingCart"
+    entity_type: "eventsourced-shopping-cart"
+    state: "Cart"
+    events: ".domain.ItemAdded"
+    events: ".domain.ItemRemoved"
+};
+  // end::proto_state[]
+// tag::service[]
   option (kalix.service) = {
     type: SERVICE_TYPE_ENTITY
     component: ".domain.ShoppingCart"
```

Comment:
The `kalix.service` annotations should be removed as well. The change is from having a `kalix.service` annotation connected to a `kalix.file` annotation with the component configuration, to just having a `kalix.codegen` annotation on the service with all the configuration.

Reply:
Wow, that was weird. I remember reproducing in local but anyways. We are good now 👍  Thanks guys
Ok. I will remove `kalix.service`